playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -55,6 +55,7 @@
   roles:
     - role: dns-views
     - role: infra-ansible/roles/dns-server
+      #when: not external_nsupdate_keys is defined
```

Comment:
this will go away once idempotency of the role verified (it shall not regen keys for consequent runs)